examples/lighting-and-environment.html

```diff
@@ -133,6 +132,148 @@ <h2>When both <span class="attribute">background-image</span> and <span class="a
+        <example-snippet stamp-to="demo-container-5" highlight-as="html">
+            <template>
+                <model-viewer controls
+                    background-color="#ff0077"
+                    environment-image="assets/whipple_creek_regional_park_04_1k.jpg"
+                    alt="A 3D model of a reflective sphere depicted within a forest"
+                    src="assets/reflective-sphere.gltf"></model-viewer>
+            </template>
+        </example-snippet>
+      </div>
+    </div>
+  </div>
+
+  <div class="sample">
+    <div class="demo" id="demo-container-6"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">6</div>
+        <h2>Supports different <span class="attribute">background-image</span> and <span class="attribute">environment-image</span> images</h2>
+        <example-snippet stamp-to="demo-container-6" highlight-as="html">
+            <template>
+                <model-viewer
+                    controls
+                    background-image="assets/whipple_creek_regional_park_04_1k.jpg"
+                    environment-image="assets/spruit_sunrise_2k.jpg"
+                    alt="A 3D model of a reflective sphere depicted within a forest"
+                    src="assets/reflective-sphere.gltf"></model-viewer>
+            </template>
+        </example-snippet>
+      </div>
+    </div>
+  </div>
+
+  <div class="sample">
+    <div class="demo" id="demo-container-7"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">7</div>
+        <h2>Control environment reflection intensity with <span class="attribute">environment-intensity</span></h2>
+        <example-snippet stamp-to="demo-container-7" highlight-as="html">
+            <template>
+              <model-viewer controls
+                    id="environment-intensity-demo"
+                    background-color="#ff0077"
+                    environment-image="assets/whipple_creek_regional_park_04_1k.jpg"
+                    alt="A 3D model of a reflective sphere depicted within a forest"
+                    src="assets/reflective-sphere.gltf"></model-viewer>
+<script>
+(() => {
+  const modelViewer = $('#environment-intensity-demo');
+  const time = performance.now();
+
+  const animate = (now) => {
+    modelViewer.environmentIntensity = oscillate(0, 6, 4000, now - time);
+    requestAnimationFrame(animate);
+  };
+
+  animate();
+})();
+</script>
+            </template>
+        </example-snippet>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="sample">
+    <div class="demo" id="demo-container-8"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="index">8</div>
+        <h2>Control stage lighting with <span class="attribute">stage-light-intensity</span></h2>
+        <example-snippet stamp-to="demo-container-8" highlight-as="html">
+            <template>
+                <model-viewer
+                    id="stage-light-demo"
```

Comment:
Ditto on some attribute defined for `stage-light-intensity`

Reply:
Fixed